src/libraries/Mono.Samples/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <RunAnalyzers>false</RunAnalyzers>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
We're still keeping the samples under `src/mono/netcore/samples`, right? (I think we should keep them, personally - they're part of the runtime team's devloop for investigating issues)

The new samples are are for CI, and we dont' expect to tweak them often?  In that case they don't need these properties - the CI samples should compile cleanly, and they don't need any unsafe blocks.

```suggestion
```

Reply:
Thanks!

Reply:
Ideally, I'd like this new spot to serve both purposes.  The goal for your inner dev loop scenario is to make them more reliable.  They had a tendency to break and we sometimes wouldn't know until later.